prac_07/myguitars.py

```diff
@@ -6,20 +6,68 @@
 """
 import csv
 from guitar import Guitar
-from operator import itemgetter
 
 
 def main():
```

Comment:
Just missing a docstring for your main function here.
The other function's docstrings are good though